testdata/src/go.uber.org/trustedfunc/testing/testing.go

```diff
@@ -1,20 +0,0 @@
-//  Copyright (c) 2023 Uber Technologies, Inc.
```

Comment:
This is actually not needed, we can just rely on stdlib `testing` library without stubs for it.